### PR TITLE
Adds a format flag when listing tokens and cluster members.

### LINKED ISSUE #161: Allow microcloud to output tables in the CLI as json and csv
Just like `lxc` can output information in different formats with the `-f` flag, this would be needed for the microcloud CLI as well.
The motivation behind this originates from the microcloud charm development. Indeed some actions like a node removal (when a charm is scaled down) require to check for the node ROLE before it actually calls `microcloud cluster remove <node_name>` .. A removal when the role is still in `PENDING` state leads to error during the removal (e.g, `Failed to run "['microcloud', 'cluster', 'remove', 'metal-3']": Error: remove /var/snap/microcloud/common/state/truststore/metal-1.yaml: file does not exist` . the certificate was probably not generated yet) 

Having a `json` output format would allow the charm developer to easily track these ROLE states and eventually enqueue an action for later if such a state like `PENDING` is detected.

### PR DESCRIPTION
**Pull request by markylaing**
Closes #161

### CODE REVIEW

**Comment by markylaing** [comment]
Blocked by #163. Will rebase once that is merged.

**Review by tomponline — APPROVED** [review]

**Pull request merged**
